integration-tests/ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,21 @@
+# Acknowledgements
+## MicroProfile Starter
+Test application ```quarkus-full-microprofile``` is a modified version of code 
+originally generated by [https://start.microprofile.io/](https://start.microprofile.io/). 
+Some utility methods were adopted from [https://github.com/eclipse/microprofile-starter/](https://github.com/eclipse/microprofile-starter/) 
+and modified to suit the purpose of this TS. The adopted work was authored 
+by Michal Karm Babacek <karm@redhat.com> and licensed under Apache 2 license.
+
+## Quarkus
+Furthermore, some parts of the TS design were derived from [https://github.com/quarkus-qe/quarkus-startstop](https://github.com/quarkus-qe/quarkus-startstop)
+The original work was authored by Michal Karm Babacek <karm@redhat.com> and licensed under Apache 2 license.
+
+## Micronaut
+Micronaut Hello World application was adapted from a generated quickstart project on [https://micronaut.io/launch/](https://micronaut.io/launch/) 
+and from the Micronaut's cli tool generated `mn create-app hello-world --build maven` project.
+
+## Helidon
+Helidon example app was adapted from a Helidon SE QuickStart archetype. See [QuickStart](https://helidon.io/docs/latest/#/se/guides/02_quickstart).
+
+## Windows console and CTRL+C
+The method of handling the CTRL+C was adopted by Michal Karm Babacek <karm@redhat.com> from hints by [Stanislav](http://stackoverflow.com/a/15281070/32453) and [rogerdpack](https://stackoverflow.com/users/32453/rogerdpack).
```

Comment:
It might be better to link to the corresponding answer/comment instead of linking to rogerdpack's profile (like you already do for Stanislav's answer.

Reply:
Ack.